spaces/viewsets.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import status
+from rest_framework import viewsets
+from rest_framework.decorators import detail_route
+from rest_framework.response import Response
+
+from api.permissions import IsPlaceOwnerOrStaff
+from .models import Place
+from .serializers import PlaceSerializer
+
+
+class PlaceViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows 'spaces' to be viewed or edited.
+    """
+    queryset = Place.objects.all()
+    serializer_class = PlaceSerializer
+
+    @detail_route(methods=['put','patch'], permission_classes=[IsPlaceOwnerOrStaff])
```

Comment:
Would it be more "cleaner" to just reimplement the "update" method instead of doing this?